### PR TITLE
Api validation error message and Location Header

### PR DESCRIPTION
**Pull request by RutsSantos**
#### Description:
Add an error dialog so the user sees when the API is down, also added a header to the Location match screen.

#### Screenshots:
Simulator Screen Shot - iPhone 11 - 2020-05-26 at 11 38 45
Simulator Screen Shot - iPhone 11 - 2020-05-26 at 15 59 53

### CODE REVIEW

File: app/views/DR/UserInfoScreen/index.js [hunk 1]
```diff
@@ -110,18 +111,48 @@ export default function UserInfo({ navigation }) {
...
 
-  let isLoading = false;
-  const disabled =
+  let disabled =
     (cid.length > 10 || passportId.length > 6 || nssId.length > 5) &&
     (phoneNumber.length > 13 || passportName.length > 8) &&
     birth
       ? false
       : true;
+  console.log(disabled);
```
Comment (inteljudajocu) [inline app/views/DR/UserInfoScreen/index.js]: remove console log

File: app/views/DR/UserInfoScreen/index.js [hunk 2]
```diff
@@ -219,18 +250,18 @@ export default function UserInfo({ navigation }) {
...
-
-                    isLoading = true;
-                    if (await sendDataToApi()) {
-                      console.log('onpress');
-                      isLoading = false;
-                      getAge(birth);
-                      navigation.navigate('Report');
-                      closeDialog(false);
+                    let response = await sendDataToApi();
+                    if (response.valid != undefined) {
```
Comment (inteljudajocu) [inline app/views/DR/UserInfoScreen/index.js]: if comparing specifically for undefined use !== instead of !=

**Review by inteljudajocu — COMMENTED** [review]
Comments

**Review by JoseFcoEstrella — APPROVED** [review]
LGTM, just solve the comments of @inteljudajocu

File: app/views/DR/UserInfoScreen/index.js (hunk 1 above)
Comment (RutsSantos) [reply app/views/DR/UserInfoScreen/index.js]: Done

File: app/views/DR/UserInfoScreen/index.js (hunk 2 above)
Comment (RutsSantos) [reply app/views/DR/UserInfoScreen/index.js]: Done

**Review by inteljudajocu — APPROVED** [review]

**Pull request merged**